web/public/js/controllers/manage-event-applications-controller.js

```diff
@@ -58,7 +58,7 @@
     }
 
     $scope.ticketTypes = [
-      {name: 'ninja', title: 'Ninja' },
+      {name: 'ninja', title: 'Youth' },
```

Comment:
consistency of name change here; same as in https://github.com/CoderDojo/cp-events-service/pull/90